lib/Frontend/FrontendInputsAndOutputs.cpp

```diff
@@ -33,12 +33,10 @@ using namespace llvm::opt;
 
 FrontendInputsAndOutputs::FrontendInputsAndOutputs(
     const FrontendInputsAndOutputs &other) {
+  clearInputs();
```

Comment:
This is a constructor, so this should already be clear, right?

Reply:
Sure. Removed it. Thanks.